pkg/sql/join.go

```diff
@@ -577,8 +580,15 @@ func (n *joinNode) Next(params runParams) (res bool, err error) {
 				continue
 			}
 			foundMatch = true
+			if n.joinType == sqlbase.JoinType_LEFT_ANTI {
+				break
```

Comment:
Add a comment to explain/remind the reader what the anti join is about, and why in the case there's a match we break out of the loop.